0-bootstrap/variables.tf

```diff
@@ -64,6 +64,18 @@ variable "skip_gcloud_download" {
   default     = true
 }
 
+variable "project_prefix" {
```

Comment:
We should override these via the cloudbuild yaml for our integration tests

Reply:
@daniel-cit already mentioned it here, so tracking https://github.com/terraform-google-modules/terraform-example-foundation/pull/289#discussion_r525124109